contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2022-03-07
+title: You can now organize your PostHog data
```

Comment:
We probably need to work on the headline a bit as this implies you couldn't organize data before.

Suggestions welcome as nothing springs to mind atm!